app/experimenter/nimbus-ui/src/components/PageResults/index.tsx

```diff
@@ -54,10 +54,12 @@ const PageResults: React.FunctionComponent<RouteComponentProps> = () => (
           </p>
           <h3 className="h6">Hypothesis</h3>
           <p>{experiment.hypothesis}</p>
-          <TableHighlights
-            primaryProbeSets={experiment.primaryProbeSets!}
-            results={analysis?.overall!}
-          />
+          {analysis && (
```

Comment:
Hrmm, do we need this check and as well as on L71 since the user should be redirected if the analysis is unavailable? Or does [this comment](https://github.com/mozilla/experimenter/blob/emtwo/more_metadata/app/experimenter/nimbus-ui/src/components/PageResults/index.tsx#L35)/something around it need any tweaking?

Reply:
Oh, no we don't need it, you're right. I was just making typescript happy, but I think `analysis!` is what we need here